### ISSUE TITLE
update release

### ISSUE THREAD

**Issue by shibumi** [morhetz/gruvbox#162]
Hello, could you please update the release? thanks! :)

**Comment by morhetz** [comment]
@shibumi sorry, not sure, I got you right
https://github.com/morhetz/gruvbox/releases this one?
you could download master as zip pkg https://github.com/morhetz/gruvbox/archive/master.zip

**Comment by shibumi** [comment]
@morhetz yes this one. I want to package it for Linux distributions therefore I need stable releases. Sorry I can't use the HEAD of master of it.

**Comment by morhetz** [comment]
@shibumi sure, np
would you mind if I release it by the end of the week as I believe it shouldn't be that urgent?

**Comment by shibumi** [comment]
@morhetz sure as you like :)
